docs/DeveloperGuide.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: Developer Guide
 ---
 * Table of Contents
-{:toc}
+  {:toc}
```

Comment:
Char, don't mind add this commit in. I think can just use this conversation to add the commit so you do not need to use IntelliJ again (hassle). This removes the TOC for DG. Np at all!
```suggestion
{:toc}
```